package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Looking good, on lines 34 and 37 of tool.js, consider using the javascript error object mentioned on FSO's assignment. It's relatively easy: `throw new Error(message)`. Also consider reviewing spaces and capitalization in the error messages, otherwise keep it up!

Reply:
k made the changes.